src/Polymorph-Widgets/UITheme.class.st

```diff
@@ -2369,6 +2381,11 @@ UITheme >> initializeForms [
 		at: #windowMenuOver put: self newWindowMenuOverForm
 ]
 
+{ #category : #accessing }
+UITheme >> labelAreaLayout [
```

Comment:
As a general comment (not about your code). I do not like that UITheme gets a setting.
The setting should put a configuration object to the UITheme which once configured should not refer to the setting object. This violates basic layering principle.

Reply:
I would prefer to have a classVariable in UITheme that holds the layout and that the settings push the correct layout object or symbol for now in that UITheme. This way we can remove Setting and UITheme should continue to work. 
This is clear not the case in the current UITheme design and it sucks!